oggm/core/inversion.py

```diff
@@ -161,6 +287,16 @@ def mass_conservation_inversion(gdir, glen_a=cfg.A, fs=0., write=True,
     fd = 2. / (cfg.N+2) * glen_a
     a3 = fs / fd
 
+    # Shape factor params
+    sf_func = None
+    if cfg.PARAMS['use_shape_factor_for_inversion'] == 'Adhikari' or \
```

Comment:
create a new variable, like ``use_shape_factor = cfg.PARAMS['use_shape_factor_for_inversion']`` and use it throughout the function to shorten the code a little bit